docs/get-started/install-conjur.md

```diff
@@ -21,24 +21,24 @@ official Conjur containers on DockerHub.
 
 1. In your terminal, download the Conjur quick-start configuration:
 
-   ```sh-session
-   $ curl -o docker-compose.yml https://www.conjur.org/get-started/docker-compose.quickstart.yml
-   ```
+{% highlight shell %}
```

Comment:
Is this a stopgap? eg should we never use "sh-session" fences in documentation, or can we introduce code to make clipboard work correctly with those?

Reply:
Yes, you can consider this a stopgap for now, just a quick way to fix it as it's urgent.